test/fixtures/test-runner/output/hooks.js

```diff
@@ -149,6 +150,55 @@ test('test hooks', async (t) => {
 });
 
 
+test('test hooks - async', async (t) => {
```

Comment:
Any reason not to add this test closer to the end of the file? It would make the diff smaller for the TAP snapshot.